python/cudf_polars/docs/overview.md

```diff
@@ -34,6 +34,8 @@ pip install --upgrade uv
 uv pip install --upgrade -r py-polars/requirements-dev.txt
 ```
 
+> ![NOTE] plain `pip install` works fine, but `uv` is _much_ faster!
```

Comment:
Is this dramatically different from `cudf_polars` than for the rest of RAPIDS? I would like to be consistent in our recommendations for installation tools across RAPIDS. If you think we should promote `uv` in the RAPIDS install guide, we can open that discussion -- but I would hesitate to recommend it for just one package.

(Note: I helped identify a bug in `uv`'s prioritization of manylinux wheels recently -- I think it's a cool tool but not at a level of long-term stability that I am ready to push it on all RAPIDS users.)

Reply:
This is copied recommendations from the polars install guide. 

I have been using some of this to test-drive some of these newer tools. I certainly will not die on this hill

Reply:
Ha! If polars is recommending it, polars users will probably be aware of uv. I’ll leave the decision to you on whether to align with polars recommendations or RAPIDS consistency.

Reply:
I will leave as is, and then we can discuss more concretely once we're building packages!